.github/workflows/docker-gdal.yml

```diff
@@ -34,7 +34,7 @@ jobs:
         run: |
           python3 -m pip install --no-cache-dir -U pip wheel
           python3 -m pip install --no-cache-dir cython~=0.29
-          python3 -m pip install --no-cache-dir --no-build-isolation -e .[dev,test,geopandas]
+          python3 -m pip install --no-cache-dir --no-build-isolation --no-use-pep517 -e .[dev,test,geopandas]
```

Comment:
Alternatively, we could also remove the `--no-build-isolation` altogether and test the default pip installation in this workflow (and then we can also remove the cython install step on the line above)

Reply:
Yeah, that's cleaner.